packages/@coorpacademy-components/src/molecule/learning-profile-radar-chart/index.js

```diff
@@ -0,0 +1,341 @@
+import React, {Fragment, useCallback, useEffect, useState} from 'react';
+import PropTypes from 'prop-types';
+import {
+  Radar,
+  RadarChart,
+  PolarGrid,
+  PolarAngleAxis,
+  ResponsiveContainer,
+  Tooltip,
+  PolarRadiusAxis
+} from 'recharts';
+import {
+  find,
+  pipe,
+  keyBy,
+  mapValues,
+  size,
+  get,
+  getOr,
+  map,
+  toPairs,
+  values,
+  isEmpty,
+  omit,
+  fromPairs,
+  times
+} from 'lodash/fp';
+import classnames from 'classnames';
+import {isMobile as isMobile_} from '../../util/check-is-mobile';
+import style from './style.css';
+
+const Gradient = ({type, colors: [firstColor, secondColor]}) => (
+  <defs>
+    <linearGradient id={`gradient-${type}`} x1="0%" y1="0%" x2="0%" y2="100%">
+      <stop offset="0%" stopColor={firstColor} />
+      <stop offset="100%" stopColor={secondColor} />
+    </linearGradient>
+  </defs>
+);
+
+const buildSvgGradient = styles =>
+  map.convert({cap: false})(({gradient}, index) => {
+    const {fill, stroke} = gradient;
```

Comment:
[style] tu peux eventuelement le destructurer dans la signature

`({gradient: {fill, stroke}}, index)=> (<svg>)`

Reply:
et utiliser la currification pour faire sauter le `styles => KEEP(THIS, styles)`